hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -128,6 +135,13 @@ spec =
 
       prop "Validate blueprint and commit transactions" $ do
         forAllBlind arbitrary $ \chainContext -> do
+          let commitSigningKey = genSigningKey `generateWith` 42
```

Comment:
What's happening here? This looks like a legitimate code not related to the move?

Reply:
@v0d1ch know what happened here?

Reply:
I think I had to move this test at certain point in time because of some dependency. I'll try to move it back to where it was originally.